File: PRAKTIKUM-PEMROGRAMAN1/MODUL2/C/PRAK205-2410817210024-MuhammadIbnuSina.c
#include <stdio.h>
#include <math.h>

int main(){
    int a,b,c,luas,keliling;

    scanf("%d %d",&a,&c);

    b=sqrt(c*c-a*a);
    keliling=a+b+c;
    luas=0.5*b*a;

    printf("Alas= %d cm\n",b);
    printf("Tinggi= %d cm\n",a);
    printf("Keliling= %d cm\n",keliling);
    printf("Luas= %d cm^2\n",luas);

    int a1,b1,c1,luas1,keliling1;

    scanf("%d",&a1);
    scanf("%d",&c1);

    b1=sqrt(c1*c1-a1*a1);
    keliling1=a1+b1+c1;
    luas1=0.5*b1*a1;

    printf("Alas= %d cm\n",b1);
    printf("Tinggi= %d cm\n",a1);
    printf("Keliling= %d cm\n",keliling1);
    printf("Luas= %d cm^2",luas1);
    return 0;
}

File: PRAKTIKUM-PEMROGRAMAN1/MODUL2/Python/PRAK205-2410817210024-MuhammadIbnuSina.py
import math

a, c = map(int, input().split())

b=math.sqrt(c*c-a*a)
keliling=a+b+c
luas=0.5*b*a

print(f"Alas= {int(b)} cm")
print(f"Tinggi= {int(a)} cm")
print(f"Keliling= {int(keliling)} cm")
print(f"Luas= {int(luas)} cm^2")

a1=int(input())
c1=int(input())

b1=math.sqrt(c1*c1-a1*a1)
keliling1=a1+b1+c1
luas1=0.5*b1*a1

print(f"Alas= {int(b1)} cm")
print(f"Tinggi= {int(a1)} cm")
print(f"Keliling= {int(keliling1)} cm")
print(f"Luas= {int(luas1)} cm^2")